code/modules/cargo/exports/sheets.dm

```diff
@@ -79,8 +79,7 @@
 	message = "of reinforced glass"
 	export_types = list(/obj/item/stack/sheet/rglass)
 
-// Bluespace Polycrystals. About as common on the asteroid as
-
+// Bluespace Polycrystals. Uncommon.
```

Comment:
Yeah. Removing a line of white space without mentioning it? They are toeing the line.